controllers/note.controllers.js

```diff
@@ -0,0 +1,27 @@
+const note = require('../models/note.model');
+const customError = require('../models/CustomError');
+const mongodb = require("../db/client.db");
+
+
+exports.create = async (req, res) => {
+
+    //Si pas de body
+    if (!req.body || !req.body.content) {
+        return res.send(new customError("Le contenu est requis", 401));
+    }
+
+    const userID = req.locals.userID;
+    const content = req.body.content;
+
+    const newNote = note.create(userID, content);
+
+    //Si pas d'erreur on rajoute la note a la BDD
+    let client = mongodb.getConnection();
+
+    try{
+        const result = await client.db("esgi").collection("note").insertOne(newNote);
+        return res.send({note: result.ops[0]});
+    }catch(e){
+        throw new customError("Impossible de rajouter la note", 402);
```

Comment:
Non, il faut retourner un message à l'utilisateur à moins que tu le fasses dans le fichier note.route.js.

Reply:
Venant de la documentation publique de fastify

Catching Uncaught Errors In Fastify
Fastify tries to catch as many uncaught errors as it can without hindering performance. This includes:

synchronous routes, e.g. app.get('/', () => { throw new Error('kaboom') })
async routes, e.g. app.get('/', async () => { throw new Error('kaboom') })
The error in both cases will be caught safely and routed to Fastify's default error handler for a generic 500 Internal Server Error response.

For customizing this behaviour, you should use setErrorHandler